shared/logger.go

```diff
@@ -54,45 +54,62 @@ func NewNilLogger() Logger {
 	return nl
 }
 
-type gaeLogger struct {
-	ctx context.Context
-}
-
-func (l gaeLogger) Debugf(format string, args ...interface{}) {
-	gaelog.Debugf(l.ctx, format, args...)
-}
+// HandleWithStandardGCL - TODO: Refactor
+func HandleWithStandardGCL(h http.HandlerFunc, project string, childLogger, parentLogger *gclog.Logger) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if project == "" || childLogger == nil || parentLogger == nil {
+			h(w, r)
```

Comment:
This case pollutes the log with `Context without logger`; [see](https://github.com/web-platform-tests/wpt.fyi/blob/42dd9f9bc9e9e34f1cfe346395e2981e9a93c373/shared/logger.go#L162)

Reply:
Will attach logrus in dev_appserver